### ISSUE TITLE
## or <# no longer working for creating help block

### ISSUE THREAD

**Issue by TMattes** [PowerShell/vscode-powershell#3763]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.

### Summary

In a PS script file before a function, when I type '##', no help block is generated anymore.
When I type '<#', the tag is immediately closed by '#>'.

A few weeks ago (before X-mas), when I typed '##', a help block was inserted.

'Powershell Help Completion' setting is set to 'BlockComment'

### PowerShell Version

```console
Name                           Value
PSVersion                      5.1.19041.1320                                                                                                                      
PSEdition                      Desktop                                                                                                                             
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1320
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Visual Studio Code Version

```console
1.63.2
899d46d82c4c95423fb7e10e68eba52050e30ba3
x64
```

### Extension Version

```console
ms-vscode.powershell@2021.12.0
```

### Steps to Reproduce

Write a PS function
Set cursor above function head
type '##' (plus enter)
-> no help block is inserted

### Logs

There are no relevant entries in the logs.

**Comment by JustinGrote** [comment]
I can't reproduce, works fine in a fresh vscode with 2021.12.0. Are you sure you're doing it within the context of a function block

https://user-images.githubusercontent.com/15258962/149184886-e6d1dab5-af25-4117-8910-8392f604f6ac.mp4

?

**Comment by TMattes** [comment]
Yes, as I always used to do  - as I said, a few weeks ago, it worked. But if you can't reproduce, there must be a problem at my side. What I already tried is to uninstall and re-install the extension, but the problem persists.
I'd like to avoid a complete re-install of VScode, because I don't want to lose all my settings. Is there a way to get another solution?

**Comment by JustinGrote** [comment]
@TMattes have you tried downloading a fresh zip of vscode and creating a "data" directory in the folder? That will create a self-contained install of vscode that you can test in. If it works there, then it's something wrong in your settings/profile/userdata/etc. and I would recommend resetting your user state and bringing back your settings slowly until you find which one is causing you the problem. Unfortunately we really can't help there.

If you are able to make this consistently repeatable on a fresh install of vscode (or after applying certain settings) where it's clearly a bug, please reopen this issue and we can look int it.

**Comment by TMattes** [comment]
@JustinGrote thanks for telling about this opportunity. I've just tried it, and I only installed the Powershell extension - unfortuantely the problem is still there. Are there some global user settings which maybe refer to both, the installed version and the copied fresh one?
I also tried it on another computer with another user, there it works, so the problem is my system or user profile, I guess. :(
Thank you for your support. :)
